packages/wallet/backend/src/rafiki/service.ts

```diff
@@ -209,14 +209,33 @@ export class RafikiService implements IRafikiService {
       source_ewallet: this.env.RAPYD_SETTLEMENT_EWALLET
     })
 
+    const user = await this.userService.getByWalletId(receiverWalletId)
+
     if (transferResult.status?.status !== 'SUCCESS') {
-      throw new Error(
-        `Unable to transfer from ${
-          this.env.RAPYD_SETTLEMENT_EWALLET
-        } into ${receiverWalletId} error message: ${
-          transferResult.status?.message || 'unknown'
-        }`
-      )
+      switch (transferResult.status?.error_code) {
+        case MessageErrorType.ERROR_MINOR_CURRENCY_UNITS:
+          if (!user)
+            throw Error('User should not be undefined for emitting socket')
+
+          await this.socketService.emitError(
```

Comment:
webhook events will retry until the payment is completed.
User will be spammed with a lot of messages in this case.

Reply:
Yes, we will not be using notification boxes to display errors, we discussed with Rico, and only the logger will receive some more details about the error, the user will not be spammed with error messages.